static/js/composebox_typeahead.js

```diff
@@ -752,7 +752,7 @@ exports.content_typeahead_selected = function (item, event) {
             beginning = beginning.substring(0, beginning.length - 2);
         }
         beginning += '#**' + item.name;
-        if (event && event.key === '>') {
+        if (this.custom_selection_state && this.custom_selection_state.key === '>') {
```

Comment:
Can't we just pass a modified event that has the old format, or even just the `.key` value?  This interface feels weird.

Reply:
I could simplify this logic for readability. I'll push back to this PR with those changes.